backend/news/240.feature

```diff
@@ -0,0 +1,2 @@
+Add content lifecycle management fields and feedback form.
+Renamed abbreviation LCM → CLM for consistency and clarity. @iFlameing
```

Comment:
```suggestion
@iFlameing
```

It's confusing to mention renaming LCM to CLM in the release notes, because this feature wasn't included in any previous release. The rename is only relevant to us developers, not users of the package.